streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalProcessorContextImplTest.java

```diff
@@ -129,4 +129,14 @@ public void shouldNotAllowToSchedulePunctuationsUsingDeprecatedApi() {
     public void shouldNotAllowToSchedulePunctuations() {
         globalContext.schedule(null, null, null);
     }
+
+    @Test(expected = UnsupportedOperationException.class)
+    public void shouldNotAllowInit() {
+        globalContext.getStateStore(GLOBAL_STORE_NAME).init(null, null);
```

Comment:
This is two methods calls, thus it should not be a one liner (test would pass if `getStateStore()` would throw `UnsupportedOperationException`.

Should be:
```
@Test
    public void shouldNotAllowInit() {
        final StateStore store = globalContext.getStateStore(GLOBAL_STORE_NAME);
        try {
            store.init(null, null);
            fail("Should have thrown UnsupportedOperationException.");
        } catch(final UnsupportedOperationException expected) { }
```

Similar below.

Reply:
Ack